skills.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "https://getdock.github.io/schemas/skills.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "User skills.",
+  "title": "User Skills",
+  "allOf": [
+    {
+      "$ref": "https://getdock.github.io/schemas/dock.json"
+    }
+  ],
+  "definitions": {
+    "skill": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "years": {
+          "type": "number"
+        },
+        "level": {
+          "type": "integer"
```

Comment:
Think we should define boundaries here. Otherwise it will be hard for other tools to make sense of unbounded integer (is it 5 stars? or is it 1-10 grade? or is it percentage?)

Reply:
Sounds good, what would be a good set of boundaries?